### ISSUE TITLE
Custom validator for comparison of two dates

### ISSUE THREAD

**Issue by link-er** [DavyJonesLocker/client_side_validations#327]
Hello
I need to add custom validator for comparing two dates - start date and end date.
I created custom validator

``` ruby
class MilestoneDatesValidator < ActiveModel::EachValidator
  def validate_each(record, attribute, value)
    if record.start_date > record.end_date
      record.errors.add(attribute, :end_date_less, options.merge(:value => value))
    end
  end
end
```

And I created ClientSideValidations custom validator. I wasn't sure how I can get another attributes values in it, but I tried to do this in such way:

``` ruby
ClientSideValidations.validators.local['milestone_dates'] = function(element, options) {
  start_date = new Date($('#milestone_start_date').val());
  end_date = new Date($('#milestone_end_date').val());
  if(end_date < start_date) {
    return options.message;
  }
}
```

But it doesn't work/ I have error only after reloading page, but not with client side validations.
I use client_side_validations (3.2.0.beta.3), client_side_validations-formtastic (2.0.0.beta.3), rails (3.2.3)

**Comment by lemeo** [comment]
Hi Bcardarella,

Sorry to bother you, I have the same question about @link-er. Could you plz tell me where is "the confirmation validator"? Is that an article or a Wiki? I can not find it. Thanks!

**Comment by tagliala** [comment]
Hi @lemeo 

The only thing I could think about is that "comparison of two dates" has been interpreted by Brian as "two date fields must have the same value", so he mentioned the confirmation validator

I'm currently using [validate_timeliness](https://github.com/adzap/validates_timeliness) to validate dates and this require a custom javascript client side validator

Please open a new issue explaining your problem and remember to meet our contributing guidelines: https://github.com/DavyJonesLocker/client_side_validations/blob/master/CONTRIBUTING.md#submitting-a-new-issue
